src/convnets/builders/mbconv.jl

```diff
@@ -1,9 +1,23 @@
-function dwsepconv_builder(block_configs, inplanes::Integer, stage_idx::Integer,
-                           width_mult::Real; norm_layer = BatchNorm, kwargs...)
+"""
+    irblockbuilder(::typeof(irblockfn), block_configs::AbstractVector{<:Tuple},
+                   inplanes::Integer, stage_idx::Integer, scalings::NTuple{2, Real};
+                   stochastic_depth_prob = nothing, norm_layer = BatchNorm,
+                   divisor::Integer = 8, kwargs...)
+
+Constructs a collection of inverted residual blocks for a given stage. Note that
+this function is not intended to be called directly, but rather by the [`mbconv_stage_builder`](@ref)
+function. This function must only be extended if the user wishes to extend a custom inverted
+residual block type.
+"""
+function irblockbuilder(::typeof(dwsep_conv_norm), block_configs::AbstractVector{<:Tuple},
+                        inplanes::Integer, stage_idx::Integer, scalings::NTuple{2, Real};
+                        stochastic_depth_prob = nothing, norm_layer = BatchNorm,
+                        divisor::Integer = 8, kwargs...)
```

Comment:
These are being used though, to pass stuff like the `se_round_fn` in for `MobileNetv3`

Reply:
I specifically mean stuff like `stochastic_depth_rate`

Reply:
Oh you mean I can simply keep them in `kwargs`?

Reply:
> I specifically mean stuff like `stochastic_depth_prob`

This particular one needs to be there because it will cause a `MethodError` for `dwsep_conv_norm` if passed through